src/encoded/docs/schema-changes.md

```diff
@@ -229,12 +306,12 @@ There are 2 categories when we talk about updating an existing schema:
 
 6. If applicable you may need to update audits on the metadata. Please refer to [making_audits]
 
-7. To document all the schema changes that occurred between increments of the ```schema_version``` update the object changelogs the **schemas/changelogs** directory. Below is an example of the changelog for above upgrade:
+7. To document all the schema changes that occurred between increments of the ```schema_version``` update the object changelogs the **schemas/changelogs** directory. Continuing with our example of upgrading genetic modifications object, the changelog for this upgrade would look like the following:
+
+        ### Schema version 7
 
-    Schema version 2
-    ----------------
+        * *purpose* property *validation* was renamed to *characterization*
```

Comment:
probably should be *purpose* property enum value *validation* was renamed (or replaced) by enum *characterization*